gumby/modules/dispersy_module.py

```diff
@@ -0,0 +1,166 @@
+from os import environ, path, getpid
+from collections import defaultdict
+from time import time
+
+from twisted.python.log import addObserver
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks
+from twisted.internet.task import deferLater
+
+from gumby.sync import experiment_callback
+from gumby.modules.experiment_module import static_module
+from gumby.modules.base_dispersy_module import BaseDispersyModule
+from gumby.modules.isolated_community_loader import IsolatedCommunityLoader
+from gumby.modules.gumby_session import GumbySession
+
+from Tribler.Core.SessionConfig import SessionStartupConfig
+from Tribler.dispersy.dispersy import Dispersy
+from Tribler.dispersy.crypto import ECCrypto, NoCrypto
+from Tribler.dispersy.endpoint import StandaloneEndpoint
+from Tribler.dispersy.util import unhandled_error_observer
+from Tribler.dispersy.candidate import CANDIDATE_STUMBLE_LIFETIME, CANDIDATE_WALK_LIFETIME, CANDIDATE_INTRO_LIFETIME
+
+
+@static_module
+class DispersyModule(BaseDispersyModule):
+    def __init__(self, experiment):
+        super(DispersyModule, self).__init__(experiment)
+        self._crypto = self._initialize_crypto()
+        self.session_id = environ['SYNC_HOST'] + environ['SYNC_PORT']
+        self.custom_community_loader = self.create_community_loader()
```

Comment:
Wasn't this already defined in the `BaseDispersyModule`? [Comment applies for all other BaseDispersyModule subclasses as well]